wire/internal/wire/wire_test.go

```diff
@@ -59,88 +61,109 @@ func TestWire(t *testing.T) {
 	}
 	wd := filepath.Join(magicGOPATH(), "src")
 
-	t.Run("Generate", func(t *testing.T) {
-		if _, err := os.Stat(filepath.Join(build.Default.GOROOT, "bin", "go")); err != nil {
-			t.Skip("go toolchain not available:", err)
-		}
-		for _, test := range tests {
-			test := test
-			t.Run(test.name, func(t *testing.T) {
-				t.Parallel()
+	// The golden directory has a dot prefix so its contents won't be found
+	// by go test.
+	goldenDir := ".golden"
```

Comment:
Would it also work to put it inside testdata? E.g., "testdata/golden/...". Or even in the same directory where the testcase is defined, e.g., "testdata/MyTest/golden.go"?